docs/source/reference.rst

```diff
@@ -7,6 +7,7 @@ Type System Reference
    :caption: Contents:
 
    stubs
+   tools
```

Comment:
I'm not 100% happy with the name "tools" as it doesn't seem to really fit the document. See below for more discussion.

Reply:
👍 I am 100% _not happy_ with it, but didn't come up with a good title. Let's figure it out!